Makefile

```diff
@@ -37,12 +37,18 @@ ORG ?= kuadrant
 # quay.io/kuadrant/limitador-operator-bundle:$VERSION and quay.io/kuadrant/limitador-operator-catalog:$VERSION.
 IMAGE_TAG_BASE ?= $(REGISTRY)/$(ORG)/limitador-operator
 
+ifeq (0.0.0,$(VERSION))
+IMAGE_TAG ?= latest
+else
+IMAGE_TAG ?= v$(VERSION)
+endif
+
 # BUNDLE_IMG defines the image:tag used for the bundle.
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
-BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:$(IMAGE_TAG)
 
 # Image URL to use all building/pushing image targets
-DEFAULT_IMG ?= $(IMAGE_TAG_BASE):latest
+DEFAULT_IMG ?= $(IMAGE_TAG_BASE):$(IMAGE_TAG)
```

Comment:
The changes in this file are setting the VERSION to the last released version, and main will no longer be set to automatically deploy the last build of main (latest). You can still deploy latest with:

```
make deploy VERSION=0.0.0
``` 

I'm not 100% sure what we want here tbh, but i feel something on the main branch needs to track the latest released version, which would normally be "VERSION". One option might be to just fix IMAGE_TAG to "latest" on master and alter on a release branch as part of a release script instead of setting it based on the VERSION being 0.0.0 or not.

Reply:
main set to latest again